lib/lhm/chunker.rb

```diff
@@ -47,15 +47,19 @@ def execute
         affected_rows = ChunkInsert.new(@migration, @connection, bottom, top, @options).insert_and_return_count_of_rows_created
         expected_rows = top - bottom + 1
 
+        Lhm.logger.info("Inserted #{affected_rows} rows into the destination table from #{bottom} to #{top}")
```

Comment:
I wouldn't log every iteration. Maybe at least once and then once every X minutes.